### DISCUSSION TITLE
Using Github Deploy Keys

### DISCUSSION THREAD

**Discussion by vin-fandemand** [fastlane/fastlane#19035] [Show and tell]
I was running ios build to publish to firebase distribution using fastlane. So I created the profiles and certificates and added to my private repository. Then I testing the fastlane script in the local machine and it works all well. I then added the script to Github CI/CD. Then I generated the private/public key withouth a passphrase and added the public key as Deploy Key in the github repository where my certificates and and private key in the Github secrets. I then referenced this private key in the Github Actions script and passed it to fastlane as an environment variable. However I am unable to clone the repo in the script. Does anyone know why and what did I miss ?

<img width="511" alt="Screen Shot 2021-06-30 at 10 32 48 AM" src="https://user-images.githubusercontent.com/64597233/123980151-508e8400-d98f-11eb-9311-f032f8d1d946.png">
<img width="615" alt="Screen Shot 2021-06-30 at 10 33 28 AM" src="https://user-images.githubusercontent.com/64597233/123980156-51271a80-d98f-11eb-97ba-09bc32e0ecb2.png">
<img width="884" alt="Screen Shot 2021-06-30 at 10 34 14 AM" src="https://user-images.githubusercontent.com/64597233/123980161-51bfb100-d98f-11eb-9bdc-5c451eda5310.png">

**Comment by crazymanish** [comment]
Hello @vin-fandemand 
please use this github-action plugin 
https://github.com/joshdholtz/fastlane-plugin-github_action
